### PR TITLE
add reference to jaas files for hbase components

### PR DESCRIPTION
**Pull request by ekund**
This fixes issue #598 .

### CODE REVIEW

File: cookbooks/bcpc-hadoop/recipes/hbase_config.rb [hunk 1]
```diff
@@ -149,9 +149,10 @@
 env_sh = {}
 env_sh[:HBASE_PID_DIR] = '"/var/run/hbase"'
 env_sh[:HBASE_LOG_DIR] = '"/var/log/hbase"'
-env_sh[:HBASE_OPTS] = '" -Djava.net.preferIPv4Stack=true -XX:+UseConcMarkSweepGC"'
+env_sh[:HBASE_OPTS] = '" -Djava.net.preferIPv4Stack=true -XX:+UseConcMarkSweepGC" -Djava.security.auth.login.config=/etc/hbase/conf/hbase-client.jaas'
```
Comment (bijugs) [inline cookbooks/bcpc-hadoop/recipes/hbase_config.rb]: Is it better if we add these env parameters when the cluster is only Kerberized like [this](https://github.com/ekund/chef-bach/blob/602d13ad822cb7f32173bdffdb8ede97c7f19522/cookbooks/bcpc-hadoop/recipes/hbase_config.rb#L85)

**Comment by ekund** [comment]
Updated as per Biju's comments.

File: cookbooks/bcpc-hadoop/recipes/hbase_config.rb [hunk 2]
```diff
@@ -168,6 +168,12 @@
   "-XX:+PrintGCApplicationStoppedTime -XX:+UseCompressedOops " + 
   "-XX:+PrintClassHistogram -XX:+PrintGCApplicationConcurrentTime"
 
+if node[:bcpc][:hadoop][:kerberos][:enable] == true then
+ env_sh[:HBASE_OPTS] = '"$HBASE_OPTS" -Djava.security.auth.login.config=/etc/hbase/conf/hbase-client.jaas'
```
Comment (amithkanand) [inline cookbooks/bcpc-hadoop/recipes/hbase_config.rb]: Should this be:
`env_sh[:HBASE_OPTS] = '"$HBASE_OPTS -Djava.security.auth.login.config=/etc/hbase/conf/hbase-client.jaas"'?

Comment (ekund) [inline cookbooks/bcpc-hadoop/recipes/hbase_config.rb]: It seems to work this way as well.  Not sure it makes a difference.

Comment (amithkanand) [inline cookbooks/bcpc-hadoop/recipes/hbase_config.rb]: I was trying it on `linux` and it didn't work for me.

**Comment by amithkanand** [comment]
LGTM

**Pull request merged**